ChronoAPI/ChronoLog/include/ChronoLogAdminRPCVisor.h

```diff
@@ -50,37 +51,33 @@ class ChronoLogAdminRPCProxy {
         return true;
     }
 
-    bool Connect(const std::string &uri, std::string &client_id) {
```

Comment:
Do you need a separate LocalConnect and Connect?

Reply:
LocalConnect means it is local, i.e. on the Visor. Connect means it is a Client call, i.e., a remote call.

Reply:
And the remote Connect RPC calls LocalConnect... LocalConnect does not make sense otherwise.